### ISSUE TITLE
Crash UE5 (5.3.2) when I open an specific project

### ISSUE THREAD

**Issue by Bluemuller** [getnamo/SocketIOClient-Unreal#428]
When I tried opening my project this crash appeared (yesterday it worked correctly)

Assertion failed: ReaderPos + Num <= ReaderSize [File:D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Public\Serialization\BufferReader.h] [Line: 52]

log report:
[CyberPunk_Street.log](https://github.com/user-attachments/files/15672214/CyberPunk_Street.log)

**Comment by getnamo** [comment]
That log doesn't have anything pointing at the plugin. You may need to catch it better (consider using visual studio debug launch)

**Comment by Bluemuller** [comment]
I closed this accidentaly 😅
I don't understand how does that works, I think that what you're talking about is when I have a project builted in C++ right? But mine is in blueprint ¿?

**Comment by Bluemuller** [comment]
Update: I made a copy of the project and navigating on folders I noticed that in the content folder appeared 2 unknown folders named __ExternalActors__ and __ExternalObjects__ I tried deleting them and after that the project opened but it started like completely empty, any ideas? 

![image](https://github.com/getnamo/SocketIOClient-Unreal/assets/171970634/9579be78-2d43-4448-8b03-a6789db382b1)

**Comment by getnamo** [comment]
Those come via world partition type maps. They're expected and if you delete them you're deleting assets from the maps.

**Comment by Bluemuller** [comment]
Fixed up. Thank you so much. The problem was that into the ExternalActors folder was a map corrupted or something so I tried deleting folders and runing UE5 until I found the one that was not working, then entered into this "corrupted"  folder and again deleting folders until I found the problematic map (there were 2 of them in diferent folders) After deleting them the problem got fixed without damaging any other map.
